spec/services/membership/withdraw_service_spec.rb

```diff
@@ -16,6 +16,20 @@
         described_class.call(membership)
       end.to change { membership.group.leader.user.notifications.count }.by -1
     end
+
+    context 'when the user has PointRequests' do
+      before(:each) do
+        evaluation = Evaluation.new(group: membership.group, semester: SystemAttribute.semester)
+        evaluation.set_default_values
+        evaluation.save!
```

Comment:
This is out-of-scope of this PR a bit, but I think this `Evaluation#set_default_values` should be private and called in `before_create` hook in `Evaluation`.